openxr/src/passthrough.rs

```diff
@@ -0,0 +1,73 @@
+use crate::{Result, Graphics, Session, sys, cvt};
+use std::ptr;
+
+/// TODO Doc
+/// TODO Link https://developer.oculus.com/documentation/native/android/mobile-passthrough/
+/// TODO Pause/unpause
+/// Create the passthrough feature.
+///
+/// Requires [`XR_FB_passthrough`](https://www.khronos.org/registry/OpenXR/specs/1.0/html/xrspec.html#XR_FB_passthrough)
+/// to be enabled.
+///
+/// See:
+/// * [`XR_FB_passthrough`](openxr/src/generated.rs:62)
+pub fn create_passthrough_fb<G: Graphics>(session: &Session<G>, flags: sys::PassthroughFlagsFB) -> Result<sys::PassthroughFB> {
```

Comment:
Similar extensions are exposed as methods on `Session`. We should be consistent.